template/template.go

```diff
@@ -421,7 +421,7 @@ func (te Expander) ExpandHTML(templateFiles []string) (result string, resultErr
 			}
 		}
 	}()
-
+	//nolint:unconvert // This ensure compatibility with previous go version
```

Comment:
Can you add a little more explanation here, e.g. some hint at what the error would be.

Reply:
I changed the description consequently